app/src/main/res/values/strings.xml

```diff
@@ -344,8 +344,8 @@
     <string name="login_help_title">Need help?</string>
     <string name="login_help_description">Jellyfin requires a server to connect with. Visit our documentation at docs.jellyfin.org or scan the QR code to get started with Jellyfin.</string>
     <string name="pref_watched_indicator">Show watched indicators</string>
-    <string name="lbl_hide_unwatched_count">Hide unwatched count</string>
-    <string name="lbl_episode_marks">On episodes only</string>
+    <string name="lbl_hide_unwatched_count">When the show has been watched</string>
```

Comment:
Using "item" instead of "show" likely makes more sense, as it also applies for other item types like movies.